File: dataset_reduzido/codenet/Java/Code/s336166208.java
public class Main {
    public static void main(String[] args) {

        int[][][] info = new int[4][3][10];
        for (int i = 0; i < 4; i ++) {
            for (int j = 0; j < 3; j ++) {
                for (int k = 0; k < 10; k ++) { info[i][j][k] = 0; }
            }
        }

        java.util.Scanner sc = new java.util.Scanner(System.in);
        int n = sc.nextInt();
        for (int i = 0; i < n; i ++) {
            int built = sc.nextInt();
            int floor = sc.nextInt();
            int room = sc.nextInt();
            int num = sc.nextInt();
            info[built - 1][floor - 1][room - 1] += num;
        }

        for (int i = 0; i < 4; i ++) {
            for (int j = 0; j < 3; j ++) {
                for (int k = 0; k < 10; k ++) {
                    System.out.print(" " + info[i][j][k]);
                }
                System.out.print("\n");
            }
            if (i != 4 - 1) { System.out.print("####################\n"); }
        }
    }
}

File: output/StarCoder-100/codenet/Java/Python/s336166208.py
info = [[[0] * 10 for i in range(3)] for j in range(4)]

n = int(input())
for i in range(n):
    built = int(input())
    floor = int(input())
    room = int(input())
    num = int(input())
    info[built - 1][floor - 1][room - 1] += num

for i in range(4):
    for j in range(3):
        for k in range(10):
            print(" " + str(info[i][j][k]), end="")
        print()
    if i!= 4 - 1:
        print("####################")
